src/utils/maker-colors.js

```diff
@@ -73,6 +73,11 @@ function isDark(hex) {
 }
 
 function enoughContrastForFill(hex1, hex2) {
+	const CONTRAST_FILL = 0.25;
+	return colord(hex1).delta(hex2) >= CONTRAST_FILL;
```

Comment:
I don't understand why we're using `colord(hex1).delta(hex2)` as the contrast check in the `enoughContrastForFill` function but `colord(hex1).contrast(hex2)` as the contrast check in the `enoughContrastForText` function. Can we standardize on one and use that for all contrast checks?

Reply:
I made them different because we primarily use the fill for solid backgrounds that have to be differentiated but note necessarily legible whereas text needs to pass a standardized WCAG contrast threshold. A color could look fine as a fill but not pass WCAG standards for text:

Primary color doesn't pass contrast, but looks fine as a fill color:
<img width="194" alt="Screen Shot 2022-10-31 at 2 05 52 PM" src="https://user-images.githubusercontent.com/1486885/199078678-bc15fd1b-1ed0-4a2e-ac4e-ce3ecc16b924.png">
<img width="193" alt="Screen Shot 2022-10-31 at 2 05 26 PM" src="https://user-images.githubusercontent.com/1486885/199078595-aaa30f9c-349a-49db-93f2-cf18fc14bc1f.png">

Reply:
Yup, I get that. My complaint has to do with using different functions for the same purpose. I would suggest renaming `CONTRAST_FILL` to `MAKER_CONTRAST_FILL` so it's clearly distinguished that this is an arbitrary threshold that we came up with and that didn't come from the WCAG and then we can do `colord(hex1).contrast(hex2) >= MAKER_CONTRAST_FILL` within `enoughContrastForFill` which would allow us to get rid of the `.delta()` method call and the `labPlugin` extension. Going from `.delta()` to `.contrast()` will probably require changing the constant value from `0.25` to something else, like `2` or `1.5` or something, but I think it'll probably work similarly to how it works now, right?

Reply:
They're not exactly the same - delta passes more colors that might have less contrast due to luminance but are visually very different based on their hues:

1. contrast at 1.25:
<img width="369" alt="Screen Shot 2022-10-31 at 2 55 43 PM" src="https://user-images.githubusercontent.com/1486885/199087765-60d3b315-202c-42ff-99e6-ebb998ea5949.png">
<img width="370" alt="Screen Shot 2022-10-31 at 2 55 53 PM" src="https://user-images.githubusercontent.com/1486885/199087791-c980749c-338e-45a8-b870-47f4e68039fe.png">

2. contrast at 1.3:
<img width="186" alt="Screen Shot 2022-10-31 at 2 48 01 PM" src="https://user-images.githubusercontent.com/1486885/199086888-b2d7dc05-8351-4a2a-84c4-ab429d2d7c63.png">
<img width="197" alt="Screen Shot 2022-10-31 at 2 48 07 PM" src="https://user-images.githubusercontent.com/1486885/199086907-d1179ee7-2821-418c-8d96-0862b535f566.png">

3. contrast at 1.5:
<img width="387" alt="Screen Shot 2022-10-31 at 3 01 56 PM" src="https://user-images.githubusercontent.com/1486885/199088888-042f66e9-0d52-4c08-b4cf-1dba360c2c05.png">
<img width="374" alt="Screen Shot 2022-10-31 at 3 02 02 PM" src="https://user-images.githubusercontent.com/1486885/199088912-ee9c7c6c-d0fb-473c-bdc6-f99137db5226.png">

4. delta at .25:
<img width="377" alt="Screen Shot 2022-10-31 at 2 56 17 PM" src="https://user-images.githubusercontent.com/1486885/199087865-61e732a6-308f-4e01-a3cf-d2c11c4a5db1.png">
<img width="368" alt="Screen Shot 2022-10-31 at 2 56 25 PM" src="https://user-images.githubusercontent.com/1486885/199087902-bf62fb56-8439-47f5-ab70-00af3d49049c.png">


If we feel strongly about not including an additional colord extension, I'm fine with going with option 3, but costs us some colors that could still work against the backgrounds.

Reply:
Thanks for explaining. If you think `delta` is worth it that's good enough for me, but can you please rename the constant `CONTRAST_FILL` to `MAKER_DELTA_FILL`? I feel like it's a lot more descriptive and follows the convention established by `WCAG_CONTRAST_TITLE` and `WCAG_CONTRAST_TEXT`, i.e. `{ORG}_{COLORD-METHOD}_{PURPOSE}`.